README.md

```diff
@@ -59,18 +59,42 @@ customElements.define('incrementer', ReactTestComponent);
 
 The key pieces of code are `... extends ReactHTMLElement` and `this.mountPoint`.
 
+> One thing to remember is that you will need to load [the webcomponentsjs polyfills](https://www.webcomponents.org/polyfills) for `ReactHTMLElement` to work in all browsers. The polyfills should be in the `<head>`, and should look something like this:
+>
+> ```html
+> <script src="https://cdnjs.cloudflare.com/ajax/libs/webcomponentsjs/2.4.3/webcomponents-bundle.js"></script>
+> <script>
+>   if (!window.customElements) {
+>     document.write('<!--');
+>   }
+> </script>
+> <script src="https://cdnjs.cloudflare.com/ajax/libs/webcomponentsjs/2.4.3/custom-elements-es5-adapter.js"></script>
```

Comment:
@Darrken I know we used `polyfill.io` in one project, and we'll need to include this in our web component builds going forward. I don't know how many other ways we potentially polyfill across our projects. Do we have a plan to consolidate how polyfills are loaded? We could potentially help with getting custom elements into `polyfill.io`, which has fallen through twice.

Reply:
I had actually hoped that whatever dependencies we were using at the time that required the polyfills would be updated so we wouldn't need to. Seems like it was one of the react-overlay depedencies. Do you recall?

Reply:
I think it's a bit harder to ship this package with the polyfill. If we have an app composed of three micro-apps loaded in with custom elements, for example, that could be three times a polyfill is loaded in that we didn't need loaded in.